operations/utility_storage/Alternate_Storage_Pools.md

```diff
@@ -0,0 +1,187 @@
+# Alternate Storage Pools
+
+## Description
+
+Creating, maintaining, and removing Ceph storage pools.
+
+## Use cases
+
+* A landing space for the CSM tarball used for upgrades.
+* Temporary space needed for maintenance or pre/post upgrade activities.
+
+## Best practices
+
+* Apply a proper quota to any pools created.
+  * This will use storage from the default CRUSH rule which is utilizing every OSD.
+  * Improper use and policing of space usage can have a negative impact on the cluster and space available to running services that have determined the space provided by the design and sizing exercises.
+* Cleanup after the criteria for the pool creation has been met.
+  * This can be as simple as removing volumes but leaving the pool for future use.
+
+## Procedures
+
+This example shows the creation and mounting of an `rbd` device on `ncn-m001`.
+
+**NOTE:** The commands to create and delete pools or `rbd` devices must be run from a master node or one of the first three storage nodes (`ncn-s001`, `ncn-s002`, or `ncn-s003`).
+
+### Create storage pool
+
+The below example will create a storage pool name `csm-release`. The pool name can be changed to better reflect any use cases outside of support for upgrades.
+
+```bash
+ceph osd pool create csm-release 3 3
```

Comment:
Do we want to give any explanation or context for what the `3 3` part of this means?

Reply:
I am torn there.  It is basically the difference between explaining in more detail what they most likely don't care about and can easily look up vs just keeping it simple and safe.  if anything we would add that into the utility storage run book for Ceph commands.  This goes back to the argument of our Ceph documentation is a lot of translating into customer friendly docs vs our k8s documentation is "go google it".